tests/saitests/py3/sai_base_test.py

```diff
@@ -235,26 +235,35 @@ def sai_thrift_port_tx_disable(self, client, asic_type, port_list, target='dst')
         else:
             sai_thrift_port_tx_disable(client, asic_type, port_list, target=target)
 
+    def get_dut_port(self, ptf_port):
```

Comment:
try and catch is not a very pythonic way of doing that. Can't we do if key exists - do1, else do2?

Reply:
Yes, fixed.